enterprise/internal/codeintel/uploads/transport/http/handler.go

```diff
@@ -63,6 +63,12 @@ func newHandler(
 	return handler
 }
 
+func IsSCIPAvailableHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	})
+}
+
```

Comment:
```suggestion
var noopHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
	w.WriteHeader(http.StatusOK)
})
```

Could make it even easier/more generic.

Reply:
Done.